src/templates/ja/2019/base.html

```diff
@@ -102,7 +102,7 @@
 {% set localizedContributors = {
   "kleinab": "<i lang=\"en\"><i lang=\"en\">Abigail Klein is a Google software engineer. She worked on Google Docs, Sheets, and Slides web accessibility where she added <a href=\"https://www.blog.google/outreach-initiatives/accessibility/whats-you-say-present-captions-google-slides/\">automatic captions to Google Slides</a>, as well as improving screen reader, braille, screen magnifier, and high contrast support. She currently works on Google Chrome and ChromeOS accessibility. She has a bachelor's and master's degree in computer science from MIT, where she co-founded an assistive technology hackathon and was a lab assistant and guest lecturer of the assistive technology class.</i>",
   "argyleink": "<i lang=\"en\">Adam Argyle is a Google Chrome developer relations member focused on CSS; He's a web addict with an insatiable lust for great UX &amp; UI; Find him on the web <a href=\"https://twitter.com/argyleink\">@argyleink</a> or checkout his website <a href=\"https://nerdy.dev\">https://nerdy.dev</a>;</i>",
-  "alankent": "<i lang=\"en\">Alan Kent is a Developer Advocate at Google focusing on e-commerce and content ecosystems. He blogs at alankent.me and tweets as @akent99.</i>",
+  "alankent": "<i lang=\"ja\">Alan KentはGoogleのDeveloper Advocateで、Eコマースとコンテンツエコシステムに焦点を当てています。彼は alankent.me でブログを書いており、@akent99 としてツイートしています。</i>",
```

Comment:
You can remove the lang parts of this. Only needed when not in the lang of the rest of the document:

```suggestion
  "alankent": "Alan KentはGoogleのDeveloper Advocateで、Eコマースとコンテンツエコシステムに焦点を当てています。彼は alankent.me でブログを書いており、@akent99 としてツイートしています",
```

Reply:
delete tag.